src/Sulu/Bundle/AdminBundle/Resources/js/components/Modal/modal.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../containers/Application/colors.scss';
+
+$backdropColor: rgba($black, .4);
+$borderRadius: 4px;
+$backgroundColor: $white;
+$modalShadow: 0 0 25px 0 rgba($black, .25);
+
+.container {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    z-index: 1;
+    pointer-events: none;
```

Comment:
`pointer-events` are set to `none` because a click on the container should behave like a click on the backdrop. However, the `pointer-events` property of the modal has to be set back to `auto`, which is missing at the time.